events_boilerplate.go

```diff
@@ -96,15 +96,6 @@ func (e *EventEditVariableType) TypeDataPtr() interface{} {
 func (e *EventEditVariablesType) TypeDataPtr() interface{} {
 	return &e.TypeData
 }
-func (e *EventNewNotificationType) TypeDataPtr() interface{} {
```

Comment:
Maybe the events can still be returned by the API and we should keep this? Or data have been migrated?